src/broker/Makefile.am

```diff
@@ -30,9 +30,18 @@ flux_broker_SOURCES = \
 	shutdown.h \
 	shutdown.c
 
+cmbd_LDFLAGS = -export-dynamic
```

Comment:
Actually, that line can just be removed now.  It was originally added when the bindings operated on only dynamic symbols.  Now that they link directly to the external interfaces it just doesn't matter anymore.